nixos/modules/services/monitoring/munin.nix

```diff
@@ -24,6 +24,10 @@ let
       logdir    /var/log/munin
       rundir    /run/munin
 
+      ${if cronCfg.extraCSS != ""
```

Comment:
How about
```
      ${lib.optionalString (cronCfg.extraCSS != "") "staticdir ${customStaticDir"}}
```
- should give the same result and be more explicit.

Reply:
Done.